Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:17.04
```

Comment:
Should be a problem but just for the record, the reference system is Debian Stretch.